proposal.md

```diff
@@ -0,0 +1,48 @@
+# Preamble
+
+Title: nft-barter Proposal document 
+
+## Abstract 
+This nft-barter is an open market where you can mint, buy, sell and swap NFTs within collection. The mint, buy and sell funtionality will be
+the standard implementation of ERC-721 with some add-ons. NFT-barter comes in where user can list item for swap, propose swap,
```

Comment:
ERC721 doesn't offer buy / sell, maybe we should only mention transfer function here

Reply:
which protocol covers that?

Reply:
I guess none, but I could be wrong. Buy and sell are market concept. ERC721 offers ownership and transfers, which can be used to implement buy / sell protocols

Reply:
@HassanAmed would you like to weigh in here?